releasenotes/notes/vqe-with-estimator-primitive-7cbcc462ad4dc593.yaml

```diff
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+     Added :class:`qiskit.algorithms.minimum_eigensolvers` package to include
+     interfaces for primitive-enabled algorithms. Refactored
+     :class:`qiskit.algorithms.minimum_eigensolvers.VQE` to leverage primitives.
+     Please see the Qiskit Runtime migration guide for more information.
```

Comment:
Unless we have some link for the migration guide - I am not sure about this. Even with this it seems we could do something more globally in a prelude somewhere around the overall move to primitives.
What might be nice here is a short working example showing the  new VQE in action that someone can see/copy etc.